test/views/TrusteesUkAddressViewSpec.scala

```diff
@@ -51,7 +51,11 @@ class TrusteesUkAddressViewSpec extends QuestionViewBehaviours[TrusteesUkAddress
       applyView,
       messageKeyPrefix,
       routes.TrusteesUkAddressController.onSubmit(NormalMode, index).url,
-      "line1", "line2", "line3", "townOrCity", "postcode"
+      Seq("line1", "line2", "line3", "townOrCity", "postcode"),
+      trusteeName
     )
```

Comment:
Do we need to pass in hint text tto assert?

Reply:
Done. Refactor of QuestionViewBehaviours.pageWithTextFields to accept an optional hit text for pageWithTextFields in order to asset